.github/workflows/README.md

```diff
@@ -136,15 +136,9 @@ using this issue [template](../nightly_failure.md).
 
 Tries to install Nilearn from wheel & check installation on all operating systems.
 
-### detect_test_pollution.yml
-
-Runs once a month.
-Use pytest with the pytest-random-order plugin to run all tests in a random order.
-This aims to detect tests that are not properly isolated from each other (test pollution).
-
 ### testing_minimum.yml
 
-Runs once a month.
+Runs and push to main and once a month.
```

Comment:
```suggestion
Runs and pushes to main, once a month.
```

Reply:
nope

but let me try to rephrase to see if this is clearer